building stopped with something related with  libarmadillo-dev ?

Hi, guys, thanks for the new code!
I have a try with the Fringe code today.  My compiling and building stopped at the Fit function. 
It seems a problem of libarmadillo-dev, which I installed with apt-get under ubuntu with anaconda3.  
Please see the following screen report. Any suggestions? thanks! 

#######################
[67%] Building CXX object src/fit/CMakeFiles/fit.dir/Fit.cc.o
/home/sun/FRINGE/src/fringe/src/fit/Fit.cc: In member function ‘virtual void Fit::estimate_Tau(str)’:
/home/sun/FRINGE/src/fringe/src/fit/Fit.cc:589:30: error: ‘index_min’ is not a member of ‘arma’
      arma::urowvec idx_min = arma::index_min(rmse_vs_tau); 
                              ^
src/fit/CMakeFiles/fit.dir/build.make:81: recipe for target 'src/fit/CMakeFiles/fit.dir/Fit.cc.o' failed
make[2]: *** [src/fit/CMakeFiles/fit.dir/Fit.cc.o] Error 1
CMakeFiles/Makefile2:506: recipe for target 'src/fit/CMakeFiles/fit.dir/all' failed
make[1]: *** [src/fit/CMakeFiles/fit.dir/all] Error 2
Makefile:148: recipe for target 'all' failed
make: *** [all] Error 2

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,7 +30,7 @@ set(CMAKE_CXX_STANDARD 11)
 FIND_PACKAGE(GDAL 3.0 REQUIRED)
 
 ###Find Armadillo
-FIND_PACKAGE(Armadillo REQUIRED)
+FIND_PACKAGE(Armadillo 8.100 REQUIRED)
 
 ###Find openmp
 find_package(OpenMP COMPONENTS CXX)

